src/main/java/com/google/sps/servlets/NewRequestServlet.java

```diff
@@ -89,10 +90,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
           request.getParameter("end-time"),
           request.getParameter("max-stops"),
           userId);
+
+      if (request.getParameter("latitude") == null || request.getParameter("latitude") == null) {
```

Comment:
Can you move these magic strings to a constant in this class?  latitude, longitude, start-address etc

Reply:
I removed them and added a DeliverySlotParameters enum.